packages.ini

```diff
@@ -27,11 +27,15 @@
 [backports-zoneinfo==0.2.1]
 python_versions = <3.9
 
+[bcrypt==4.0.0]
+
 [beautifulsoup4==4.7.1]
 [beautifulsoup4==4.11.1]
 
 [billiard==3.6.4.0]
 
+[black==21.11b1]
+validate_extras = d
```

Comment:
I would try upgrading ops -- this is a pretty old version of `black` with the annoying `click` bug

Reply:
i vaguely remember there was a reason we were click<8.0, i'll revisit it